reason-parser/src/reason_parser.mly

```diff
@@ -1200,7 +1201,7 @@ _use_file:
 parse_core_type:
     core_type EOF
       {
-        let core_loc = mklocation $startpos($1) $endpos($1) in
+        let core_loc = mklocation $symbolstartpos $endpos($1) in
```

Comment:
Some extra background: I changed all the `$startpos($1)` to `$symbolstartpos` as that seems to be the one we want to have always anyway.